web/pages/profile.tsx

```diff
@@ -112,6 +113,12 @@ export default function ProfilePage(props: {
       //eslint-disable-next-line @typescript-eslint/ban-ts-comment
       // @ts-ignore
       twitchInfo: deleteField(),
+      //eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      initialIpAddress: deleteField(),
```

Comment:
I'm not sure I think this is a good idea. What about trolls that come onto the site, wreak havoc with spam, then go and delete their account - how can we ban their ip address going forward?